src/App.scss

```diff
@@ -10,14 +11,13 @@ html, body {
   nav {
     background-color: #ddd;
     color: #222;
-    padding: .5%;
+    padding: 0.5%;
     a {
       padding-right: 1rem;
     }
   }
 
   .component {
-    height: 85%;
+    height: 130%;
```

Comment:
I am not sure if we should use this approach, when looking at my own screen it also stretches the visualization. We may have to see if we can dynamically determine the right height of the visualization/toolbar.